data/mods/gen1jpn/README.md

```diff
@@ -0,0 +1,55 @@
+Gen 1, the beginning
```

Comment:
This should probably be deleted. It's just a copy/paste from Gen 1, right? I don't see anything specific to Japanese Gen 1.

Reply:
There is some changes in terms of moves from international Gen 1 like Blizzard freezing 30% of the time, draining moves always missing vs substitutes, and Swift not always hitting, but outside of that they are the same mechanically. I could replace it with a list of the differences between Japanese Gen 1 and International Gen 1.

Reply:
Just implemented this.